### PR TITLE
[DO-1826] Download fields relationship_type and relationship_type_en from pep.org.ua

### PR DESCRIPTION
**Pull request by Andrii17s**

### CODE REVIEW

File: business_register/models/pep_models.py [hunk 1]
```diff
@@ -264,6 +264,8 @@ class CompanyLinkWithPep(DataOceanModel):
                                                       'Can be: bank_customer, owner, manager, by_position, other.')
     relationship_type = models.CharField(_("connection`s type"), max_length=550, null=True,
                                          help_text='Type of connection between the person and this company')
+    relationship_type_en = models.CharField(_("connection`s type in English"), max_length=550, null=True,
```
Comment (roman-tiukh) [inline business_register/models/pep_models.py:267]: localization?

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Review by roman-tiukh — COMMENTED** [review]
I confirm the PR but I will merge it a little later

**Comment by roman-tiukh** [comment]
we are waiting for second approve

File: business_register/converter/pep.py [hunk 2]
```diff
@@ -531,20 +538,26 @@ def save_or_update_peps_companies(self, peps_companies_data):
...
+                                                      source_id, relationship_type, relationship_type_en)
                     is_changed = True
                 else:
                     update_fields = []
                     if already_stored_link.category != category:
                         already_stored_link.category = category
                         update_fields.append('category')
+                    if already_stored_link.relationship_type != relationship_type:
+                        already_stored_link.relationship_type = relationship_type
+                        update_fields.append('relationship_type')
```
Comment (IegorSoboliev) [inline business_register/converter/pep.py:557]: in all cases you should update also 'updated_at' field

**Review by IegorSoboliev — CHANGES_REQUESTED** [review]

File: business_register/converter/pep.py (hunk 2 above)
Comment (roman-tiukh) [reply business_register/converter/pep.py:557]: ?

Comment (Andrii17s) [reply business_register/converter/pep.py:557]: `
if update_fields:                        update_fields.append('updated_at')                        already_stored_link.save(update_fields=update_fields)                        is_changed = True
`
Line 576

**Review by EmmaSamoilova — APPROVED** [review]

**Review by IegorSoboliev — APPROVED** [review]

File: business_register/converter/pep.py (hunk 2 above)
Comment (IegorSoboliev) [reply business_register/converter/pep.py:565]: ! Yes, pardon. Forgot my own code)))

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**